src/Views/FirmwareView.vala

```diff
@@ -96,9 +105,17 @@ public class About.FirmwareView : Gtk.Stack {
     }
 
     private void add_device (Fwupd.Device device) {
-        if (device.has_flag (Fwupd.DeviceFlag.UPDATABLE) && device.releases.length () > 0) {
+        if (device.has_flag (Fwupd.DeviceFlag.UPDATABLE)) {
             var row = new Widgets.FirmwareUpdateRow (fwupd, device);
-            update_list.add (row);
+
+            int position = -1;
+            if (device.releases.length () > 0 && device.latest_release.version != device.version) {
+                num_updatable_devices++;
+                position = 1;
+            }
+            update_row_headers ();
+
+            update_list.insert (row, position);
```

Comment:
Instead of this, there should be a https://valadoc.org/gtk+-3.0/Gtk.ListBox.set_sort_func.html